core-bundle/src/Resources/contao/modules/ModuleArticleList.php

```diff
@@ -102,7 +102,7 @@ protected function compile()
 			(
 				'link' => $objArticles->title,
 				'title' => StringUtil::specialchars($objArticles->title),
-				'id' => $cssID[0] ?: 'article-' . $objArticles->id,
+				'id' => $cssID[0] ?? 'article-' . $objArticles->id,
```

Comment:
Wouldn’t this change the behavior if `$cssID[0]` is set to the empty string?

```suggestion
				'id' => ($cssID[0] ?? null) ?: 'article-' . $objArticles->id,
```